Formula/file-roller.rb

```diff
@@ -51,6 +69,6 @@ def post_install
   end
 
   test do
-    system "#{bin}/file-roller", "--version"
+    assert_match version.to_s, shell_output("#{bin}/file-roller --version 2>/dev/null")
```

Comment:
😆 You'll be hard pressed to find GTK-based applications that don't throw errors similar to this. I think it's okay without personally.